### ISSUE TITLE
'High Contrast Black' theme button active state color

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#995]
**Describe the bug**
Button's active(pressed) state text color and the background color is the same: white

> What version of the Fundamental Library Styles are you using?
v0.9.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/button.html
2. Click on any button

**Expected behavior**
The text color should be black.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81629072-1e9b1680-93b7-11ea-9cc0-7efd4dd22ec9.png)
![image](https://user-images.githubusercontent.com/43764832/81630152-b13cb500-93b9-11ea-82b5-4a12035d3a44.png)
![image](https://user-images.githubusercontent.com/43764832/81630206-d2050a80-93b9-11ea-918a-4b73d75b8bd7.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by droshev** [comment]
this has been addressed with the delta theming
